src/components/ButtonGroupPicker/index.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import StyledContainer from './styled/container';
+import StyledButtonGroup from './styled/buttonGroup';
+import StyledLegend from './styled/label';
+import { uniqueId } from '../../libs/utils';
+import { Provider } from './context';
+import RenderIf from '../RenderIf';
+import HelpText from './styled/helpText';
+import ErrorText from './styled/errorText';
```

Comment:
since HelpText and ErrorText are the same in all inputables we can reuse the ones defined inside Input component in
`src/components/Input/styled/`